sql/core/src/test/scala/org/apache/spark/sql/CachedTableSuite.scala

```diff
@@ -93,13 +93,24 @@ class CachedTableSuite extends QueryTest with SQLTestUtils with SharedSQLContext
     }.sum
   }
 
+  // Blocking uncache table for tests
+  private def uncacheTable(tableName: String): Unit = {
```

Comment:
My only concern is whether a user would ever need this behavior as well, but I think it's pretty specific to our tests. I'm OK with it